lib/workers/global/config/parse/env.ts

```diff
@@ -38,6 +38,7 @@ const renameKeys = {
   aliases: 'registryAliases',
   azureAutoComplete: 'platformAutomerge', // migrate: azureAutoComplete
   gitLabAutomerge: 'platformAutomerge', // migrate: gitLabAutomerge
+  recreateClosed: 'recreateWhen',
```

Comment:
Isn't this field meant for direct renames, not renames+transforms?

Reply:
It works but it is not fine. I will create a new array for transforms+migrations and put it there.